src/proto/Filecoin.proto

```diff
@@ -30,4 +30,5 @@ message SigningInput {
 // Transaction signing output.
 message SigningOutput {
     bytes encoded = 1;
```

Comment:
should we always keep one for consistency? whichever is accepted by the RPC endpoint?

Reply:
done